reports/wip-limits.ts

```diff
@@ -39,24 +63,25 @@ function getWipViolationIcon(limit: number, actual: number): string {
     }
 }
 
-export function render(projData: ProjectData): string {
+export function render(reportData: any): string {
```

Comment:
So if this takes the WipData map declared above then render is just iterating over those keys with `for in`